concourse/steps/scan_container_images.py

```diff
@@ -211,3 +213,23 @@ def result_to_tuple(upload_result: UploadResult):
       tablefmt='html',
     )
     return table
+
+
+def create_license_report(license_report):
```

Comment:
separate to own commit - feel free to merge to master already (the entire move, of course)